README.md

```diff
@@ -1,14 +1,14 @@
-# stactools-package
+# stactools-ephemeral
```

Comment:
I agree that `ephemeral` shouldn't get us into any trouble w.r.t. renaming via scripts, but I'm a little worried about how this README looks to someone coming to the template repo. I think if I saw this title I'd be a bit confused. Some options off the top of my head:
1. Add a top section to the README with the template info, with instructions to delete
2. Have a main `README.md` that's for the template repo w/ how-to-use instructions, and a second README-ephemeral that gets moved to README as part of the rename process

Not sure what the answer is, but I think we do want the landing page to be a bit friendlier?

Reply:
Agreed. I'll add a quick note at the top for now. A separate README is probably a good idea.

Reply:
I've added a separate shorter README that gets replaced by the templated one during `rename`.